util/usm_helper.h

```diff
@@ -21,8 +21,12 @@ namespace usm_helper {
  */
 template <sycl::usm::alloc alloc, typename elems_typeT>
 auto allocate_usm_memory(const sycl::queue &queue, size_t num_elements = 1) {
-  const auto &device{queue.get_device()};
   const auto &context{queue.get_context()};
+  // According to the SYCL 2020 the first device in context should be
+  // used for alloc::host
```

Comment:
```suggestion
  // used for alloc::host.
```

Reply:
this comment has been applied in this commit: b8c581fe5eb0ed09444a50db00f9cc6c718a154a